internal/gensnippets/gensnippets.go

```diff
@@ -42,44 +43,61 @@ import (
 
 // Generate reads all modules in rootDir and outputs their examples in outDir.
 func Generate(rootDir, outDir string, apiShortnames map[string]string) error {
+	blankDir := []string{rootDir}
+	if err := GenerateSnippetsDirs(rootDir, outDir, apiShortnames, blankDir); err != nil {
+		return err
+	}
+	return nil
```

Comment:
nit: just return calling the function. Also no need to declare the variable, just inline that.

Reply:
Also instead of passing a value for this case pass nil. If nil is passed do the old logic. Should streamline code a bit.